lua/impatient/cachepack.lua

```diff
@@ -1,16 +1,26 @@
 local ffi = require('ffi')
 
 -- using double for packing/unpacking numbers has no conversion overhead
-local sizeof_c_double = ffi.sizeof("double")
-local c_double = ffi.typeof("double[1]")
+if jit.arch == "arm" then
```

Comment:
A descriptive comment about this supporting 32 bit systems would help. It's not immediately obvious that `arm` is 32 bit.